aerleon/lib/nokiasrl.py

```diff
@@ -62,9 +83,20 @@ class SRLTerm(openconfig.OCTerm):
 
     def SetAction(self) -> None:
         action = self.ACTION_MAP[self.term.action[0]]
-        self.term_dict['action'] = {action: {}}
+        log = {}
+        if self.term.logging:
+            if action == 'drop':
+                log = {"log": True}
+            else:
+                raise UnsupportedLogging(
+                    'logging can only be used with deny in term %s' % self.term.name
```

Comment:
You should try and use f-strings, they are much more readable. This would be f'logging can only be used with deny in term {self.term.name}'

Reply:
done